packages/atomic/tailwind.config.js

```diff
@@ -39,6 +39,7 @@ module.exports = {
         lg: 'var(--atomic-text-lg)',
       },
       screens: {
+        desktop: {min: '640px'},
```

Comment:
This enables `@screen desktop`

Reply:
Would it be possible to use the `sm` prefix for the 640px breakpoint? e.g. `@screen sm`
Or is the prefix only for classes?

Also, looking at the line below with `mobile` set to just 1px less, I'm wondering if we are following the tailwind practice of writing mobile-first styles, with overrides for larger screens? It seems strange to have two screens defined separated by 1px.

Something like this might be more aligned:

```
.density-normal {
    ... mobile styles first

    @screen desktop {
        ... desktop overrides
    }

}
```



Ref: https://tailwindcss.com/docs/responsive-design#mobile-first

Reply:
> Would it be possible to use the `sm` prefix for the 640px breakpoint? e.g. `@screen sm`
Or is the prefix only for classes?

This prefix is already used by default in tailwind and is a minimum screen size rather than a maximum. Since the two definitions I added are meant to be mutually exclusive and used in fewer contexts, I don't want to overwrite the default. I'll rename them to add `-only` suffixes.

> Also, looking at the line below with `mobile` set to just 1px less, I'm wondering if we are following the tailwind practice of writing mobile-first styles, with overrides for larger screens? It seems strange to have two screens defined separated by 1px.

For most components I would prefer following the mobile-first approach recommended by Tailwind. IMO, this case is exceptional because desktop and mobile have almost completely different properties, making it much easier to maintain as two separate files that never override eachother.

Reply:
I just noticed that I made a mistake with the breakpoint, it should be `768px` or `1024px`, not `640px`.